src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -381,6 +382,9 @@ public boolean validateAndProcessDataImport(ODMContainer odmContainer, String st
                                     eventCrf = updateEventCrf(eventCrf, userAccount, Status.AVAILABLE, null);
                                 }
 
+                            }else if (itemCountInForm.getInsertedUpdatedItemCountInForm() == 0 && eventCrf.getAnnotations().equals(NEW_CRF_TRACK)) {
+                            	// remove the new created blank Form
+                            	eventCrfDao.remove(eventCrf);
```

Comment:
Instead of creating even_crf and then deleting it from the database, try to create the event_crf only when there is a valid item data

Reply:
done